queue_services/entity-filer/src/entity_filer/filing_processors/continuation_out.py

```diff
@@ -0,0 +1,66 @@
+# Copyright © 2023 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""File processing rules and actions for the continuation out filing."""
+from contextlib import suppress
+from datetime import datetime
+from typing import Dict
+
+import dpath
+from legal_api.models import Business, Comment, Filing
+
+from entity_filer.filing_meta import FilingMeta
+from entity_filer.filing_processors.filing_components import filings
+
+
+def process(business: Business, continuation_out_filing: Filing, filing: Dict, filing_meta: FilingMeta):
+    """Render the continuation out filing into the business model objects."""
+    # update continuation out, if any is present
+    with suppress(IndexError, KeyError, TypeError):
+        court_order_json = dpath.util.get(filing, '/continuationOut/courtOrder')
+        filings.update_filing_court_order(continuation_out_filing, court_order_json)
+
+    continuation_out_json = filing['continuationOut']
+
+    details = continuation_out_json.get('details')
+    legal_name = continuation_out_json.get('legalName')
+    continuation_out_date = datetime.strptime(continuation_out_json.get('continuationOutDate'), '%Y-%m-%d')
```

Comment:
I think the better approach would be 
```
continuation_out_date = continuation_out_json.get('continuationOutDate')
continuation_out_date = LegislationDatetime.as_utc_timezone(datetime.fromisoformat(continuation_out_date))
```
This way we don't have to add time manually. The other one still work since we don't care about time here

Reply:
@kzdev420 can you update the processor and test to use the approach Vysakh provided.

Reply:
Yes, I will